Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/deploymentslot/DeploymentSlotNode.java

```diff
@@ -128,8 +125,7 @@ private void delete() {
     @AzureOperation(name = "webapp|deployment.swap", params = {"this.slot.name()", "this.webApp.name()"}, type = AzureOperation.Type.ACTION)
     private void swap() {
```

Comment:
can this method be removed?

Reply:
This is the action method for deployment slot node, which is required for now.